docs/recovery.md

```diff
@@ -0,0 +1,351 @@
+# Table of Contents
+
+* [Recovering Funds From `lnd` (funds are safu!)](#recovering-funds-from-`lnd`-(funds-are-safu!))
+  * [On-Chain Recovery](#on-chain-recovery)
+    * [24-word Cipher Seeds](#24-word-cipher-seeds)
+    * [Wallet+Seed Passphrases](#wallet+seed-passphrases)
```

Comment:
TOC link is broken, along with 4 others

Reply:
Link should be `#walletseed-passphrases`.